tests/components/history/test_init.py

```diff
@@ -628,3 +628,25 @@ def set_state(entity_id, state, **kwargs):
         "/api/history/period/{}".format(dt_util.utcnow().isoformat())
     )
     assert response.status == 200
+
+
+async def test_fetch_period_api_with_include_order(hass, hass_client):
```

Comment:
Just to confirm - this is a test for something else, right? not the introduced functionality?  I don't see it validating the results, other than HTTP status.

Reply:
The request failed before my fix, because it assumed each entry for an entity had at least 1 state. It does indeed not validate the results.